connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignorTest.java

```diff
@@ -76,23 +83,21 @@
 
     @Parameters
     public static Iterable<?> mode() {
-        return Arrays.asList(new Object[][] {{CONNECT_PROTOCOL_V1, CONNECT_PROTOCOL_V2}});
+        return Arrays.asList(new Object[][] {{CONNECT_PROTOCOL_V1}, {CONNECT_PROTOCOL_V2}});
```

Comment:
This can be simplified into `return Arrays.asList(new Object[] {CONNECT_PROTOCOL_V1, CONNECT_PROTOCOL_V2});` but since you're removing it in https://github.com/apache/kafka/pull/11983, we can ignore it.

Reply:
Blegh, thanks. I'll simplify it anyways; rather leave `trunk` in a healthy place and not risk FUD from someone else copy+pasting this style.